tests/codeceptjs/editors/rating_test.js

```diff
@@ -4,9 +4,22 @@ Feature('rating');
 
 Scenario('should be disabled if "readonly" is specified', async (I) => {
   I.amOnPage('read-only.html');
-  assert.equal(await I.grabAttributeFrom('[id="root[rating]5"]', 'disabled'), 'true');
-  assert.equal(await I.grabAttributeFrom('[id="root[rating]4"]', 'disabled'), 'true');
-  assert.equal(await I.grabAttributeFrom('[id="root[rating]3"]', 'disabled'), 'true');
-  assert.equal(await I.grabAttributeFrom('[id="root[rating]2"]', 'disabled'), 'true');
-  assert.equal(await I.grabAttributeFrom('[id="root[rating]1"]', 'disabled'), 'true');
+  //assert.equal(await I.grabAttributeFrom('[id="root[rating]5"]', 'disabled'), true);
+  //assert.equal(await I.grabAttributeFrom('[id="root[rating]4"]', 'disabled'), true);
+  //assert.equal(await I.grabAttributeFrom('[id="root[rating]3"]', 'disabled'), true);
+  //assert.equal(await I.grabAttributeFrom('[id="root[rating]2"]', 'disabled'), true);
+  //assert.equal(await I.grabAttributeFrom('[id="root[rating]1"]', 'disabled'), true);
+
+  // Fails because the input element is not visible (display: none)
+  //I.seeElement('[id="root[rating]5"]:disabled');
+  //I.seeElement('[id="root[rating]4"]:disabled');
+  //I.seeElement('[id="root[rating]3"]:disabled');
+  //I.seeElement('[id="root[rating]2"]:disabled');
+  //I.seeElement('[id="root[rating]1"]:disabled');
```

Comment:
Comments should be removed, applies to all commented code

Reply:
I just kept them as reference while testing since I had to commit new test changes to see if the tests passed on WebDriver/Selenium. I'll cleanup everything in the final commit.

I have added some extra helpers for testing if an element is disabled or readonly.
 * **I.amDisabled(xpath)**
 * **I.amNotDisabled(xpath)**
 * **I.amReadOnly(xpath)**
 * **I.amNotReadOnly(xpath)**

If any of these functions fail, the session will terminate with an error.

#### Functions that extends default functions
 * **I.pressKey(key)** - Extended so Puppeteer will press sequence of character if "Unknown key" like WebDriver
* **I.amAcceptingPopups()** - Extended so WebDriver will accept the Puppeteer command
* **I.amCancellingPopups()** - Extended so WebDriver will accept the Puppeteer command

#### Misc (Currently not used but might be useful later.)
* **grabBooleanAttributeFrom(xpath, attrib)** - Get real Boolean value from attribbute.
* **isTrue(val)** - Returns real Boolean value